src/status_im/contexts/communities/events.cljs

```diff
@@ -478,3 +508,28 @@
 (rf/reg-event-fx :communities/update-last-opened-at-success
  (fn [{:keys [db]} [community-id last-opened-at]]
    {:db (assoc-in db [:communities community-id :last-opened-at] last-opened-at)}))
+
+(rf/defn share-community
+  {:events [::share-community-confirmation-pressed]}
+  [cofx users community-id]
+  (when (seq users)
+    {:json-rpc/call [{:method      "wakuext_shareCommunity"
+                      :params      [{:communityId community-id
+                                     :users       (vec users)}]
+                      :js-response true
+                      :on-success  #(rf/dispatch [::people-invited %])
+                      :on-error    #(do
```

Comment:
Please, refer to the Clojure Style guide here https://guide.clojure.style/#no-multiple-forms-fn-literals

Reply:
Used `(fn [] ...)`